PHPCI/Logging/Handler.php

```diff
@@ -33,30 +53,39 @@ class Handler
     );
 
     /**
-     * @var LoggerInterface
+     * @var Logger
      */
     protected $logger;
 
     /**
-     * @param LoggerInterface $logger
+     * @var boolean
      */
-    public function __construct(LoggerInterface $logger = null)
+    protected $registered = false;
+
+    /**
+     * @param Logger $logger
+     */
+    public function __construct(Logger $logger = null)
```

Comment:
Nope: \Monolog\Logger, because pushHandler (line 87) is a method of \Monolog\Logger.